### PR TITLE
Fix/refinement issues

### PR DESCRIPTION
**Pull request by Alejoboga20**
Description:
Fixing styles issues found it after refinement.
Disabling images in home page (client is asking to only activate sport) And making sport image clickable.
Editing Item cards based on new requirements.
Adding Go Back in dolphin logo.

Reviewers:
@natokra @kevin-rodriguez @ImanolH96 @Alejoboga20 

Screenshots:
image
image.

### CODE REVIEW

File: src/infrastructure/services/nft/NftService.ts [hunk 1]
```diff
@@ -7,7 +7,8 @@ const nftApi = api.injectEndpoints({
     getNfts: builder.mutation<NFT[], string>({
       query: issuer => `${endpoints.NFT}?issuer=${issuer}`,
       transformResponse: (data: { result: NFT[] }) => {
-        return data.result.slice(0, 25);
+        const filterArray = data.result.filter(nft => nft.offerPrice === 25.0);
+        return filterArray;
```
Comment (natokra) [inline src/infrastructure/services/nft/NftService.ts:11]: WDYT about returning the value this way?

```suggestion
        return data.result.filter(nft => nft.offerPrice === 25.0);   
```

**Review by natokra — APPROVED** [review]
just a minor suggestion 😄

File: src/infrastructure/services/nft/NftService.ts (hunk 1 above)
Comment (Alejoboga20) [reply src/infrastructure/services/nft/NftService.ts:11]: nice, I will do that

File: src/infrastructure/services/nft/NftService.ts [hunk 2]
```diff
@@ -4,10 +4,10 @@ import { NFT } from 'app/interfaces/NFT/NFT';
 
 const nftApi = api.injectEndpoints({
   endpoints: builder => ({
-    getNfts: builder.mutation<NFT[], string>({
+    getNftsWithFilter: builder.mutation<NFT[], string>({
       query: issuer => `${endpoints.NFT}?issuer=${issuer}`,
       transformResponse: (data: { result: NFT[] }) => {
-        return data.result.slice(0, 25);
+        return data.result.filter(nft => nft.offerPrice === 25.0);
       },
```
Comment (natokra) [inline src/infrastructure/services/nft/NftService.ts:11]: WDYT about doing it this way? sorry I just saw it clearer

```
 transformResponse: (data: { result: NFT[] }) => 
       result.filter(nft => nft.offerPrice === 25.0),
```

**Pull request merged**